backend/src/middleware/checkers.ts

```diff
@@ -31,14 +32,11 @@ export const checkAdmin: AuthHandler = async (req, _res, next) => {
 // any routes that uses this NEEDS a registryId param
 export const checkOwnership: AuthHandler = async (req, _res, next) => {
   try {
-    const { registryId } = req.params;
```

Comment:
Could you revert all this back to before.

This middleware function checks if a user owns the registry that they are changing,